CHANGELOG.md

```diff
@@ -41,6 +41,8 @@
     the direct cookie-secret to bytes without silent padding added.
 
 ## Changes since v5.1.1
+
+- [#548](https://github.com/oauth2-proxy/oauth2-proxy/pull/548) Move logging options out of main package (@JoelSpeed)
```

Comment:
This isn't quite correct. Shouldn't it be "Splitting logging options from global options structure"

Reply:
Rebase problems 😅  Will fix